Update the documentation with the new product names (#517)
suse-edge/suse-edge.github.io: asciidoc/components/longhorn.adoc

:warning-caption: :warning:
endif::[]

SUSE Storage is a lightweight, reliable, and user-friendly distributed block storage system designed for Kubernetes. It is a product based on Longhorn, an open-source project initially developed by Rancher Labs and currently incubated under the CNCF.

== Prerequisites

If you are following this guide, it assumes that you have the following already available:

* At least one host with SUSE Linux Micro 6.0 installed; this can be physical or virtual
* A Kubernetes cluster installed; either K3s or RKE2
* Helm

== Manual installation of SUSE Storage

=== Installing Open-iSCSI

> A core requirement of deploying and using SUSE Storage is the installation of the `open-iscsi` package and the `iscsid` daemon running on all Kubernetes nodes.

Reviewer: Mixing up product and component names here. Applies to many references within this file. Which is the correct one to use?

> This is necessary, since SUSE Storage relies on `iscsiadm` on the host to provide persistent volumes to Kubernetes.

Reviewer: ```suggestion
This is necessary, since Longhorn relies on `iscsiadm` on the host to provide persistent volumes to Kubernetes.
```

[TIP]
====

> For additional help installing open-iscsi, refer to the {link-longhorn-iscsi}[official SUSE Storage documentation].

Reviewer: ```suggestion
For additional help installing open-iscsi, refer to the {link-longhorn-iscsi}[official Longhorn documentation].
```

helm repo update
----
+

> . Install SUSE Storage in the `longhorn-system` namespace:

Reviewer: ```suggestion
. Install Longhorn in the `longhorn-system` namespace:
```

[TIP]
====
The concept of storage in Kubernetes is a complex, but important topic. We briefly mentioned some of the most common Kubernetes resources,

> however, we suggest to familiarize yourself with the {link-longhorn-terminology}[terminology documentation] that SUSE Storage offers.

Reviewer: ```suggestion
however, we suggest to familiarize yourself with the {link-longhorn-terminology}[terminology documentation] that Longhorn offers.
```
